### ISSUE TITLE
Show menu when right click on asset list item & preview asset (if possible) on double click

### ISSUE THREAD

**Issue by ColinTree** [mit-cml/appinventor-sources#2054]
I'm talking about this menu:
![image](https://user-images.githubusercontent.com/22613139/75103027-316c3d80-5649-11ea-89ba-baa2e89230f9.png)

Codes at around https://github.com/mit-cml/appinventor-sources/blob/c5b3e2d22cb942d5b13e4fce7fce2236ff5f7467/appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java#L98

Double clicking detect would a little bit hard without a built-in "DoubleClickHandler"..

**Comment by bhoomika01** [comment]
Right now when we click on the asset it shows the menu . 
so it is suppose to show when we right click the asset item
@ColinTree 
Am i right?

**Comment by ColinTree** [comment]
@bhoomika01  Yes I think that is doable.

**Comment by bhoomika01** [comment]
@ColinTree  I would like to try this

**Comment by ColinTree** [comment]
@bhoomika01 That's surely ok for me, but I recommend you to wait for further suggestions from @ewpatton .

**Comment by bhoomika01** [comment]
@ColinTree Ya sure.

**Comment by ewpatton** [comment]
I think the double-click idea is compelling. But I fail to understand why right click is necessary given that the menu already shows on left click?

**Comment by bhoomika01** [comment]
@ewpatton Should we add the feature to preview asset item on double click ?

**Comment by ewpatton** [comment]
@bhoomika01 Yes, if you want to go ahead and implement the double-click functionality that's fine.

**Comment by bhoomika01** [comment]
I've implemented double click functionality.
@ewpatton 
![bandicam-2020-04-02-22-31-54-476](https://user-images.githubusercontent.com/58931490/78278719-163bfa00-7534-11ea-96bb-ea3df4abc1b0.gif)

**Comment by bhoomika01** [comment]
#2145

**Comment by bhoomika01** [comment]
#2145 Modified AssetList.java to preview asset on double click.
